CONTRIBUTING.md

```diff
@@ -1,20 +1,34 @@
 # Contributing to the Apache Cassandra GoCQL Driver
 
-**TL;DR** - this manifesto sets out the bare minimum requirements for submitting a patch to gocql.
+**TL;DR** This manifesto sets out the bare minimum requirements for submitting a patch to gocql. It also offers some suggestions to speed up the review, approve and merge process.
 
 This guide outlines the process of landing patches in gocql and the general approach to maintaining the code base.
 
 ## Background
 
-The goal of the gocql project is to provide a stable and robust CQL driver for Go.  This is a community driven project that is coordinated by a small team of developers in and around the Apache Cassandra project.  For security, governance and administration issues please refer to the Cassandra Project Management Committee.
+The goal of the gocql project is to provide a stable and robust CQL driver for Go. This is a community driven project that is coordinated by a small team of developers in and around the Apache Cassandra project. For security, governance and administration issues please refer to the Cassandra Project Management Committee.
+
+## Engage with the community early
+
+If you are interested in contributing a particular feature or bug fix we heavily encourage you to start a discussion with the community or at the very least announce your interest in working on it before jumping right into writing code. It helps reduce the likelihood of multiple people working on the same issue in parallel when they could be collaborating instead. Getting feedback early in the contribution process will also greatly speed up the review, approval and merge process. 
+
+Common ways to engage with the GoCQL community are: 
+
+- CASSGO project on ASF JIRA
```

Comment:
Please add hyperlinks so it is easier to find the resources.

Reply:
Hmm I'm not sure what hyperlink I should use for the ML, I think I'll use the archives link (https://lists.apache.org/list.html?dev@cassandra.apache.org)

Reply:
done